requirements.txt

```diff
@@ -0,0 +1,3 @@
+nibabel>=4.0
```

Comment:
It looks like the line adding the affine property dates back 9 years (from the tags, looks like version 2.0.0):

https://github.com/nipy/nibabel/commit/11151705c87d3c54bb3dc635df77ad8f25f72818#diff-bf1768d37a4ffcc36b8398d4d8b04b13ef1f492cfc371e84516eff208a9283b7R335

4.0.0 is the absolute newest release, from june 2022.  Should we loosen this version requirement so people are more likely to be able to use it without excess changes in an existing python environment?

Reply:
It seems like the numpy deprecation was only fixed in nibabel version [3.2.1](https://github.com/nipy/nibabel/releases/tag/3.2.1). That version thus seems to be the earliest version which is compatible with the latest version of numpy. I will update the requirements file to match that.